src/IR/terminals.ts

```diff
@@ -45,6 +45,12 @@ export interface Text<Value extends string = string> extends BaseNode {
   readonly value: Value;
 }
 
+export type IDCastable = string | Identifier;
```

Comment:
Should this be `= string | Identifier<true, string>`?

Reply:
No, this is for arbitrary identifiers, including names of defined functions.